x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/logs/logs_tab_content.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import type { Filter } from '@kbn/es-query';
+import { EuiFlexGroup, EuiFlexItem, EuiLoadingSpinner } from '@elastic/eui';
+import { SnapshotNode } from '../../../../../../../common/http_api';
+import { LogStream } from '../../../../../../components/log_stream';
+import { useHostsViewContext } from '../../../hooks/use_hosts_view';
+import { useUnifiedSearchContext } from '../../../hooks/use_unified_search';
+import { useLogsSearchUrlState } from '../../../hooks/use_logs_search_url_state';
+import { LogsLinkToStream } from './logs_link_to_stream';
+import { LogsSearchBar } from './logs_search_bar';
+
+export const LogsTabContent = () => {
```

Comment:
While testing the Logs tab, I wondered whether we should re-render and fetch new logs every time a user reopens the Logs tab.

E.g:

https://user-images.githubusercontent.com/2767137/227182915-d8dcdf8b-6f49-4f1a-9652-958f7bede06d.mov


That means if the search time range is set with relative time ranges (E.g: 15m-now), each time the user opens the Logs tab, it will make a new request, and new entries will be added.

I don't know if that will be confusing from the analysis standpoint since, currently, the page only reloads the other tabs/components when a new search is submitted. I also found it a bit annoying having to wait for logs that were previously loaded to be loaded again if I reopen the tab.

I'm curious about your opinion on it